src/modules/storefront/provider.tsx

```diff
@@ -1,35 +1,27 @@
-import React, { useEffect, useState } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import { initArweave } from '@/modules/arweave';
 import arweaveSDK from '@/modules/arweave/client';
 import { isNil } from 'ramda';
 import { Storefront } from '@/modules/storefront/types';
-import { Wallet } from '@/modules/wallet/types';
 import { StorefrontContext } from './context';
+import { useWallet } from '@solana/wallet-adapter-react';
 
-type StorefrontProviderChildrenProps = {
-  searching: boolean;
-  storefront?: Storefront;
-};
-
-type StorefrontProviderProps = {
-  wallet?: Wallet;
-  children: (props: StorefrontProviderChildrenProps) => React.ReactElement;
-};
-
-export const StorefrontProvider = ({ wallet, children }: StorefrontProviderProps) => {
+export const StorefrontProvider: FC = ({ children }) => {
```

Comment:
good opportunity to switch to our new pattern

```suggestion
export function StorefrontProvider(props: {children: React.ReactElement}): JSX.Element {
```